### PR TITLE
Support multiline text in plugin configuration secret field

### PR DESCRIPTION
**Pull request by karolinakuzniewicz**
I want to merge this change because...

SALEOR-1577

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1577 Dashboard should support multiline text field in the plugin configuration](https://app.clickup.com/t/2549495/SALEOR-1577)

**Review by krzysztofwolski — APPROVED** [review]

File: src/plugins/components/PluginSecretFieldDialog/PluginSecretFieldDialog.tsx [hunk 1]
```diff
@@ -63,6 +63,10 @@ const PluginSecretFieldDialog: React.FC<PluginSecretFieldDialogProps> = ({
           <>
             <DialogContent>
               <TextField
+                multiline={
+                  maybe(() => field.type) ===
```
Comment (jwm0) [inline src/plugins/components/PluginSecretFieldDialog/PluginSecretFieldDialog.tsx:67]: Use optional chain operator instead

**Review by jwm0 — APPROVED** [review]

File: src/plugins/components/PluginSecretFieldDialog/PluginSecretFieldDialog.tsx (hunk 1 above)
Comment (karolinakuzniewicz) [reply src/plugins/components/PluginSecretFieldDialog/PluginSecretFieldDialog.tsx:67]: done

File: schema.graphql [hunk 2]
```diff
@@ -4,28 +4,40 @@ schema {
 }
 
 type AccountAddressCreate {
-  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  errors: [Error!]!
```
Comment (krzysztofwolski) [inline schema.graphql:7]: Please do not reformat schema file. Just copy from Core repo or edit needed lines.

Comment (karolinakuzniewicz) [reply schema.graphql]: done

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**